Formula/threesdk.rb

```diff
@@ -0,0 +1,31 @@
+class Threesdk < Formula
+  desc "Software development kit for JumpscaleX"
+  homepage "https://wiki.threefold.io/"
+  url "https://github.com/threefoldtech/jumpscaleX_core/archive/v10.5.2.tar.gz"
+  sha256 "d2ce77a61a198b2cd53f4acce5fd7e5c36c369fc83a05b9f74314530b4f4a175"
+
+  bottle do
```

Comment:
New formula should not have a bottle block.

Reply:
Removed now